### ISSUE TITLE
AZ CLI [Extension 0.4.2] Vsix publish error in Node

### ISSUE THREAD

**Issue by carlosscastro** [Microsoft/botbuilder-tools#819]
Tom tried publishing a node.js bot and got this.

```bash
C:\scratch\TestBot\nodebot\nodeBot>az bot publish --resource-group CloneBot105 -n CloneBot105 --subscription 9ad09adf-c85f-43a0-96fe-18b146d1f90e -v v4 --verbose --code-dir "."

The behavior of this command has been altered by the following extension: botservice
Preparing Bot Builder SDK v4 bot for publish, with code directory . and project file .
Detected bot language Node, Bot Builder version v4.
Creating upload zip file.
Creating upload zip file, code directory ..
Adding node_modules to folders to exclude from zip file.
Compressing bot source into C:\scratch\TestBot\nodebot\nodeBot\upload.zip.
Zip file path created, at C:\scratch\TestBot\nodebot\nodeBot\upload.zip.
Bot source published. Preparing bot application to run the new source.
Detected language javascript. Installing node dependencies in remote bot.

'KuduClient' object has no attribute '__initialized'
Traceback (most recent call last):
  File "C:\Users\VSSADM~1\AppData\Local\Temp\pip-install-e5ce5s7r\knack\knack\cli.py", line 206, in invoke
  File "C:\Users\VSSADM~1\AppData\Local\Temp\pip-install-e5ce5s7r\azure-cli-core\azure\cli\core\commands\__init__.py", line 346, in execute
  File "C:\Users\VSSADM~1\AppData\Local\Temp\pip-install-e5ce5s7r\six\six.py", line 693, in reraise
  File "C:\Users\VSSADM~1\AppData\Local\Temp\pip-install-e5ce5s7r\azure-cli-core\azure\cli\core\commands\__init__.py", line 320, in execute
  File "C:\Users\VSSADM~1\AppData\Local\Temp\pip-install-e5ce5s7r\azure-cli-core\azure\cli\core\commands\__init__.py", line 169, in __call__
  File "C:\Users\VSSADM~1\AppData\Local\Temp\pip-install-e5ce5s7r\azure-cli-core\azure\cli\core\__init__.py", line 440, in default_command_handler
  File "C:\Users\tomlm\.azure\cliextensions\botservice\azext_bot\custom.py", line 92, in publish_app
    __install_node_dependencies(kudu_client)
  File "C:\Users\tomlm\.azure\cliextensions\botservice\azext_bot\custom.py", line 28, in __install_node_dependencies
```

**Comment by carlosscastro** [comment]
@stevengum to update after he tries to repro node. Otherwise, potentially Tom's environment may be missing the latest core cli.

**Comment by stevengum** [comment]
Reproduced, looking into fix

**Comment by stevengum** [comment]
Have a fix, will cut private bits for testing.

**Comment by stevengum** [comment]
Fixed via Azure/azure-cli-extensions#443.
